package.json

```diff
@@ -160,7 +160,7 @@
     "eslint-plugin-react-hooks": "^1.6.0",
     "eslint-plugin-scanjs-rules": "^0.1.4",
     "eslint-plugin-sonarjs": "^0.5.0",
-    "eslint-plugin-va-enzyme": "./script/custom-eslint/va-enzyme",
+    "eslint-plugin-va": "file:./script/eslint-plugin-va",
```

Comment:
I don't have a preference of using `file:` or not. Any suggestions here?

Reply:
as long as it works

Reply:
If I'm reading the [docs](https://eslint.org/docs/developer-guide/working-with-plugins#configs-in-plugins) correctly, I think ESLint uses `plugin` instead of `file`. But ultimately, I agree with Rian. I don't think it matters as long as it works.

Reply:
I removed it and it works